sonar-dotnet-shared-library/src/main/java/org/sonarsource/dotnet/shared/plugins/AbstractPropertyDefinitions.java

```diff
@@ -70,7 +70,7 @@ public AbstractPropertyDefinitions(String languageKey, String languageName, Stri
         .category(languageName)
         .defaultValue(fileSuffixDefaultValue)
         .name("File suffixes")
-        .description("Comma-separated list of suffixes of files to analyze.")
+        .description("List of suffixes for files to analyze.")
```

Comment:
```suggestion
        .description("List of suffixes of files to analyze.")
```

(the suffix of a file)

Reply:
I didn't like the `of..of` repetition. It's `of..for` in half of other plugins:

https://github.com/search?p=2&q=org%3ASonarSource+%22List+of+suffixes%22&type=Code